src/mdformat/_conf.py

```diff
@@ -59,6 +60,9 @@ def _validate_values(opts: Mapping, conf_path: Path) -> None:  # noqa: C901
     if "end_of_line" in opts:
         if opts["end_of_line"] not in {"crlf", "lf", "keep"}:
             raise InvalidConfError(f"Invalid 'end_of_line' value in {conf_path}")
+    if "validate" in opts:
+        if not isinstance(opts["validate"], bool):
+            raise InvalidConfError(f"Invalid 'validate' value in {conf_path}")
```

Comment:
These lines aren't being recorded as tested in coverage, but they are being exercised by the addition to `test_invalid_conf_value`

Do you have any advice for what I might be missing?

Reply:
From CI:
```
 Name                                                                                                Stmts   Miss Branch BrPart  Cover   Missing
-----------------------------------------------------------------------------------------------------------------------------------------------
/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/mdformat/_conf.py      59      0     44      1    99%   64->66
-----------------------------------------------------------------------------------------------------------------------------------------------
```

It seems it's missing branch coverage jumping from line 64 to 66. So we need a test that has _valid_ value for `validate` option in the TOML file.

Reply:
Oh that makes sense!